### PR TITLE
Trip results and itinerary styling

### PR DESCRIPTION
**Pull request by hbruch**
## Proposed Changes

  - Styling for pages trip-results und and trip-itinerary
  - replacing #333 and #333333 in scss files by $black (which as default has #333) 

## Pull Request Check List

  - A reasonable set of unit tests is included
  - Console does not show new warnings/errors
  - Changes are documented or they are self explanatory
  - This pull request does not have any merge conflicts
  - All existing tests pass in CI build
  - Code coverage does not decrease (unless measured incorrectly)

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform
  - Check that the implementation matches the design, when such one is defined in a Jira issue
  - Merge the pull request

### CODE REVIEW

File: app/component/EndLeg.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ function EndLeg(props, context) {
         </div>
       </div>
       <div className={`leg-before ${modeClassName}`} aria-hidden="true">
-        <div className={`leg-before-circle circle ${modeClassName}`} />
+        <div className={`leg-before-circle el circle ${modeClassName}`} />
```
Comment (derhuerst) [inline app/component/EndLeg.js]: Is this on purpose? If "el" is supposed to signify "end leg", let's use `end-leg`.

File: app/component/SummaryPage.js [hunk 2]
```diff
@@ -2450,6 +2450,9 @@ class SummaryPage extends React.Component {
               />
             </div>
           );
+          // EMBARK override content with activeTab only
+          content = itineraryTabs[activeIndex];
```
Comment (derhuerst) [inline app/component/SummaryPage.js:2454]: Would it work to just call the `selectedItineraries` map function with `selectedItiniraries[activeIndex]` & `activeIndex`, in order not to render all other tabs? We would still keep the diff against upstream relatively small.

File: app/component/EndLeg.js (hunk 1 above)
Comment (hbruch) [reply app/component/EndLeg.js]: No it is not, wil remove it again

**Review by derhuerst — CHANGES_REQUESTED** [review]
While testing the changes locally via http://localhost:8080/summary/OKC%20MUSEUM%20OF%20ART%20(3000)%3A%3A35.469801%2C-97.519528/NW%2023RD%20%40%20MERIDIAN%20(199)%3A%3A35.493284%2C-97.601012?time=1678041095, I noticed some visually broken elements:

Screenshot 2023-03-05 at 19 33 59
Screenshot 2023-03-05 at 19 34 10
Screenshot 2023-03-05 at 19 34 28

**Pull request merged**